docs/tutorials-and-quickstart-guides/openms-user-tutorial.md

```diff
@@ -18,48 +18,44 @@ OpenMS User Tutorial
 ### Installation
 
 Before we get started, we will install OpenMS and KNIME. If you take part in a training session you will have likely received an USB stick from us that contains the required data and software. If we provide laptops with the software you may of course skip the installation process and continue reading the next section.
+If you are doing this tutorial online, choose online in the following tab(s).
 
-#### Installation from the OpenMS USB stick
+::::{tab-set}
+
+:::{tab-item} Online
+:sync: online
+
+If you are working through this tutorial at home/online, proceed with the following steps:
+
+- Download and install OpenMS using the installation instructions for your operating system:
+  - [GNU/Linux](/installations/installation-on-gnu-linux.md)
+  - [macOS](/installations/installation-on-macos.md)
+  - [Windows](/installations/installation-on-windows.md)
+- Download and install [KNIME](https://www.knime.org/downloads/overview)
+
+:::
+
+:::{tab-item} USB Stick
+:sync: usb
 
 Please choose the directory that matches your operating system and execute the installer.
 
-For Windows, you call:
+For Windows, you run:
 
 - The OpenMS installer: {{ '{path}'+'`Windows,OpenMS-{0}-Win64.exe`'.format(version) }}
 - The KNIME installer: {{ '{path}'+'`Windows,KNIME-{0}-Installer-64bit.exe`'.format(knime_version) }}
-- OpenMS prerequisites (Windows-only): After installation, before your first use of the OpenMS plugin in KNIME, you will be asked to download it automatically if certain requirements are not found in your Windows registry. Alternatively, you can get a bundled version here or on the OpenMS USB stick: {{ '{path}'+'`Windows,OpenMS-{0}-prerequisites-installer.exe`'.format(version) }}.
```

Comment:
After completing the installation, when using the OpenMS plugin in KNIME for the first time, you may be prompted to download it automatically if specific requirements are not found in your Windows registry.